controller-server/src/main/java/com/yahoo/vespa/hosted/controller/restapi/application/ApplicationApiHandler.java

```diff
@@ -2546,14 +2543,16 @@ private long asLong(String valueOrNull, long defaultWhenNull) {
         }
     }
 
+    /*
     private void toSlime(Run run, Cursor object) {
```

Comment:
Remove?

Reply:
Sure, will do later. Wasn't sure it was unused, because things didn't compile at the time XD

Reply:
Done